### ISSUE TITLE
Add support for modifiers, key, phase and other attributes on handler tag.

### ISSUE THREAD

**Issue by ar5had** [bgrins/xbl-analysis#12]
Currently the converter just ignores the modifier attributes.
example - https://searchfox.org/comm-central/source/mail/base/content/glodaFacetBindings.xml#42

**Comment by bgrins** [comment]
Yeah, I just bumped into this with trees in https://bugzilla.mozilla.org/show_bug.cgi?id=1523957. I manually fixed it up using `event.getModifierState("Accel")`, etc.

Remaining consumers in m-c: https://searchfox.org/mozilla-central/search?q=modifiers%3D&path=xml

**Comment by ar5had** [comment]
I used `event.shiftKey` to check for shift key. 

There is a case to catch tho.
XBL Handler with shift modifier will not get executed for `opt + shift + tab`, but if you just check via using event.shfitKey for `opt + shift + tab`case then the case is truthy so the js handler might get executed if you dont' take care of this case. This might not be something to worry about but this is  one difference that I found.
